docker-compose.yml

```diff
@@ -1,79 +1,82 @@
-trustroots:
-  container_name: trustroots
-  restart: always
-  build: .
-  # Mount the app dir in the container so our changes
-  #  to the app code are also changed in the container
-  #
-  # Mount node_modules separately as a data volume, so
-  #  that they won't get overwritten by app directory
-  #  Read more: http://stackoverflow.com/a/32785014/1984644
-  volumes:
-   - ./:/trustroots
-   - /trustroots/node_modules
-   - /trustroots/public
-   - /trustroots/uploads
-  command: npm run start:docker
-  links:
-    - mongodb:mongodb
-    - maildev:maildev
-    - influxdb:influxdb
-  ports:
-    - "3000:3000"
-    - "5858:5858"
-    - "35729:35729"
-  environment:
-    - NODE_ENV=development
-    - DB_1_PORT_27017_TCP_ADDR=mongodb
-    - PORT=3000
-    - DOMAIN=trustroots.dev
-    - VIRTUAL_HOST=localhost,trustroots.dev
-    - VIRTUAL_PORT=3000
+version: "3"
 
-## SMTP Server + Web Interface for viewing and testing emails during development.
-## http://djfarrelly.github.io/MailDev/
-maildev:
-  image: djfarrelly/maildev
-  container_name: maildev
-  restart: always
-  ports:
-    - "1080:1080"
+networks:
+  trustroots:
 
-## Database container
-mongodb:
-  image: mongo:3.6
-  container_name: db_1
-  restart: always
-  command: "--smallfiles --logpath=/dev/null"
-  ports:
-    - "27017:27017"
+volumes:
+  node-modules:
+  public:
+  uploads:
 
-## InfluxDB container
-## https://github.com/tutumcloud/influxdb
-influxdb:
-  image: tutum/influxdb:latest
-  container_name: influxdb
-  restart: always
-  ports:
-    - "8083:8083"
-    - "8086:8086"
-  environment:
-    - ADMIN_USER=root
-    - INFLUXDB_INIT_PWD=root
-    - PRE_CREATE_DB=trustroots
+services:
+  trustroots:
+    container_name: trustroots
+    restart: always
+    build: .
+    # Mount the app dir in the container so our changes
+    #  to the app code are also changed in the container
+    #
+    # Mount node_modules separately as a data volume, so
+    #  that they won't get overwritten by app directory
+    #  Read more: http://stackoverflow.com/a/32785014/1984644
+    networks:
+      - trustroots
+    volumes:
+      - ./:/trustroots
+      - node-modules:/trustroots/node_modules
+      - public:/trustroots/public
+      - uploads:/trustroots/uploads
+    command: npm run start
```

Comment:
This now means it runs `maildev` inside the trustroots container _aswell_ as the one in the seperate container. The `start:docker` command is basically the same, except that it doesn't run `maildev`.

I'm pretty agnostic on whether it's better to run it in it's own container, or inside the trustroots container, but it should only run in one.

**if running in trustroots container**

- remove the `maildev` container
- move the mapped port to the trustroots container
- make sure the local.docker.js config stuff reflects the change too (I had some more thoughts on this I'll write seperately, as it could be for another PR)
- remove "start:docker" script definition from package.json

**if running standalone**

- either switch back to `start:docker` command...
- ...or could implement an environment variable to disable starting maildev (e.g. something like `"dashboard:mail": "[ -n \"$DISABLE_MAILDEV\" ] || maildev",`, although that's a bit ugly imo...)

Reply:
I have done that.

> trustroots@0.3.3 install-dependencies /trustroots
> node ./bin/install-deps.js

internal/modules/cjs/loader.js:960
  throw err;
  ^

Error: Cannot find module '/trustroots/bin/install-deps.js'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:957:15)
    at Function.Module._load (internal/modules/cjs/loader.js:840:27)
    at Function.executeUserEntryPoint [as runMain] (internal/modules/run_main.js:74:12)
    at internal/main/run_main_module.js:18:47 {
  code: 'MODULE_NOT_FOUND',
  requireStack: []
}

I don't feel very qualified to fuck around with node (or whatever this all is...)

Reply:
Done. We'll know if it's good once the one below is resolved. I couldn't get to that step

Reply:
I removed the separate container for Maildev from Docker compose. It's heavy handed to run it in its own container if it can simply run together with the app — and that's where its version is already maintained. Yay for no duplication. :-)